Intermittent failure on ImportMeasuresTests.test_import_measures_practice_statistics

The test suite has been intermittently (but quite regularly, for me) failing with this error:
```
======================================================================
FAIL: test_import_measures_practice_statistics (frontend.tests.commands.test_import_measures_new.ImportMeasuresTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/code/openprescribing/frontend/tests/commands/test_import_measures_new.py", line 415, in test_import_measures_practice_statistics
    self.validate_practice_statistics_measure_value(mv, ccgs.loc[mv.pct_id])
  File "/code/openprescribing/frontend/tests/commands/test_import_measures_new.py", line 587, in validate_practice_statistics_measure_value
    self.assertAlmostEqual(mv.percentile, series['ratio_percentile'])
AssertionError: 35.4838709677419 != 34.67741935483871 within 7 places
```

Failed again on the same line, but the precise figures are different:
```
======================================================================
FAIL: test_import_measures_practice_statistics (frontend.tests.commands.test_import_measures_new.ImportMeasuresTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/code/openprescribing/frontend/tests/commands/test_import_measures_new.py", line 415, in test_import_measures_practice_statistics
    self.validate_practice_statistics_measure_value(mv, ccgs.loc[mv.pct_id])
  File "/code/openprescribing/frontend/tests/commands/test_import_measures_new.py", line 587, in validate_practice_statistics_measure_value
    self.assertAlmostEqual(mv.percentile, series['ratio_percentile'])
AssertionError: 22.5806451612903 != 23.387096774193548 within 7 places
```

And again:
```
======================================================================
FAIL: test_import_measures_practice_statistics (frontend.tests.commands.test_import_measures_new.ImportMeasuresTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/code/openprescribing/frontend/tests/commands/test_import_measures_new.py", line 415, in test_import_measures_practice_statistics
    self.validate_practice_statistics_measure_value(mv, ccgs.loc[mv.pct_id])
  File "/code/openprescribing/frontend/tests/commands/test_import_measures_new.py", line 587, in validate_practice_statistics_measure_value
    self.assertAlmostEqual(mv.percentile, series['ratio_percentile'])
AssertionError: 9.67741935483871 != 8.870967741935484 within 7 places
```
BQ dataset: https://bigquery.cloud.google.com/dataset/ebmdatalabtest:measures_0708

--- a/openprescribing/frontend/tests/commands/test_import_measures_new.py
+++ b/openprescribing/frontend/tests/commands/test_import_measures_new.py
@@ -179,7 +179,7 @@ class ImportMeasuresTests(TestCase):
 
         df['items'] = numerators.groupby(org_column)['items'].sum()
         df['thousand_patients'] = (
-            denominators.groupby(org_column)['total_list_size'].sum() / 1000
+            denominators.groupby(org_column)['thousand_patients'].sum()
         )
         df['ratio'] = df['items'] / df['thousand_patients']
         df['items'] = df['items'].fillna(0)
@@ -203,7 +203,7 @@ class ImportMeasuresTests(TestCase):
 
         df['cost'] = numerators.groupby(org_column)['actual_cost'].sum()
         df['thousand_patients'] = (
-            denominators.groupby(org_column)['total_list_size'].sum() / 1000
+            denominators.groupby(org_column)['thousand_patients'].sum()
         )
         df['ratio'] = df['cost'] / df['thousand_patients']
         df['cost'] = df['cost'].fillna(0)
@@ -584,7 +584,7 @@ def upload_practice_statistics():
                     'ccg_id': practice.ccg_id,
                     'stp_id': practice.ccg.stp_id,
                     'regional_team_id': practice.ccg.regional_team_id,
-                    'total_list_size': total_list_size,
+                    'thousand_patients': total_list_size / 1000.0,
                 },
                 ignore_index=True,
             )

